tests/requirements.txt

```diff
@@ -1,3 +1,3 @@
 pytest===6.1.2
-pytest-asyncio==0.14.0
-websockets==8.1
+pytest-asyncio==0.20.2
+websockets==10.4
```

Comment:
Looks like this change broke e2e tests.

Reply:
I'll revert the websockets change only. Let's see if the `pytest-asyncio` one is fine.